### ISSUE TITLE
which one is the checkpoint?

### ISSUE THREAD

**Issue by tangyuelm** [deepmind/kinetics-i3d#116]
Thanks for your codes and model. I want to download the i3d model pre-trained on the Kinetics dataset but feel confused about the checkpoint. I followed the path in evaluate_sample.py
_CHECKPOINT_PATHS = {
    'rgb': 'data/checkpoints/rgb_scratch/model.ckpt',
    'rgb600': 'data/checkpoints/rgb_scratch_kin600/model.ckpt',
    'flow': 'data/checkpoints/flow_scratch/model.ckpt',
    'rgb_imagenet': 'data/checkpoints/rgb_imagenet/model.ckpt',
    'flow_imagenet': 'data/checkpoints/flow_imagenet/model.ckpt',
}
I copied the checkpoints folder to a new path and tried to load it.
However, is seems that no such file or directory: 'checkpoints/rgb_scratch_kin600/model.ckpt'
I found 4 files under the folder rgb_scratch_kin600 but cannot find the model.ckpt. Could anyone help me?
Besides, can the checkpoint be loaded to model writing in Pytorch?

**Comment by tangyuelm** [comment]
To my understand, the rgb_scratch_600 means the checkpoints for RGB frames pre-trained on Kinetics-600. What do the folder rgb_scratch and flow_scratch mean?
Besides, in each folder, there are four files called "checkpoints", "model.ckpt.data-00000-of-00001", "model.ckpt.index", "model.ckpt.meta", but there is no file called model.ckpt. Which one should I use? 
I am not familiar with Tensorflow, so I am wondering how to load these checkpoints in Pytorch. Do you have the .pth version for the checkpoint pre-trained on Kinetics-400? I can only find the checkpoints pre-trained on ImageNet and Charades in your Pytorch codes from https://github.com/piergiaj/pytorch-i3d, but I believe Kinetics-400 is more commonly used for pre-training.

**Comment by Tortoise17** [comment]
I am also looking for same answer. Or any way to convert them to .h5 format?

**Comment by joaoluiscarreira** [comment]
Sorry for the long delay. rgb_scratch means model trained from scratch on
pixels (RGB values). From scratch as opposed, to from ImageNet-pretrained
weights. Flow_scratch means models trained from scratch on optical flow.

I'm not familiar with Pytorch either... i'm sure there are Pytorch
checkpoints on the web, though.

Best,

Joao

**Comment by joaoluiscarreira** [comment]
Sorry, I'm not entirely sure about PyTorch. Do look online for Pytorch
checkpoints, there should be some available.

Best,

Joao

**Comment by Tortoise17** [comment]
@joaoluiscarreira  thank you so much. Is there any command line with which I can input the mp4 and get back the logits and features? or do I have to export flow and frames for the input in model to get these.

**Comment by BlackTea-c** [comment]
> To my understand, the rgb_scratch_600 means the checkpoints for RGB frames pre-trained on Kinetics-600. What do the folder rgb_scratch and flow_scratch mean? Besides, in each folder, there are four files called "checkpoints", "model.ckpt.data-00000-of-00001", "model.ckpt.index", "model.ckpt.meta", but there is no file called model.ckpt. Which one should I use? I am not familiar with Tensorflow, so I am wondering how to load these checkpoints in Pytorch. Do you have the .pth version for the checkpoint pre-trained on Kinetics-400? I can only find the checkpoints pre-trained on ImageNet and Charades in your Pytorch codes from https://github.com/piergiaj/pytorch-i3d, but I believe Kinetics-400 is more commonly used for pre-training.


is there the pytorch version of i3d? https://github.com/piergiaj/pytorch-i3d

**Comment by joaoluiscarreira** [comment]
Hi,

scratch means trained from scratch, without imagenet pretraining. For
tensorflow inference I'd recommend loading models from tf-hub as that's
more user friendly. This repository is a bit old unfortunately.

There are several pytorch versions of i3d, if you google around.

Best,

Joao
